### ISSUE TITLE
IcSelect with searchable prop opens IcMenu by default on render

### ISSUE THREAD

**Issue by MI6-651** [mi6/ic-ui-kit#591]
## Summary of the bug
When conditionally rendering the IcSelect (searchable), the menu will open by default on second render.

Additionally the focus state does not render

## 🧐 Expected behaviour
The menu should be closed by default.
